config/locales/en.yml

```diff
@@ -147,24 +157,17 @@ en:
         supplier_number:
           attributes:
             supplier_number:
-              invalid_format: "LGFS supplier number invalid format"
-              not_unique: "LGFS supplier number has already been taken"
+              invalid: invalid format
```

Comment:
Is this added to a description string?

Reply:
Yes, the old behaviour overrode the default Rails behaviour for error messages. This one just sets the locale for the default error code when the `format` is invalid.